tests/end-to-end/external-solution-integration/pkg/testsuite/register_application_in_compass.go

```diff
@@ -8,6 +8,7 @@ import (
 	"github.com/avast/retry-go"
 	"github.com/kyma-incubator/compass/components/director/pkg/graphql"
 	acClient "github.com/kyma-project/kyma/components/application-operator/pkg/client/clientset/versioned/typed/applicationconnector/v1alpha1"
+	esClient "github.com/kyma-project/kyma/components/event-sources/client/generated/clientset/internalclientset/typed/sources/v1alpha1"
```

Comment:
[Package names should be lowercase](https://rakyll.org/style-packages/#lowercase-only) (the one above is a bad example)